app/src/main/res/drawable/thumb_netcup.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="454dp"
+    android:height="454dp"
+    android:viewportWidth="454"
+    android:viewportHeight="454">
```

Comment:
As above, please decrease size to at most 128x128

Reply:
Changed thumbnail size to 128x128 in ca70d29